blueprints/ember-frost-list/index.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   afterInstall: function () {
     return this.addAddonsToProject({
       packages: [
-        {name: 'ember-frost-theme', target: '^1.3.0'}
+        {name: 'ember-frost-core', target: '0.0.13 - 1'}
```

Comment:
This seems a bit specific, and also doesn't seem like a valid version. Is that the same as `0.0.13-1`? and if so, what's with 1) the pinned version within a component and 2) the `-1`?